Meta/Lagom/CMakeLists.txt

```diff
@@ -172,7 +172,11 @@ install(
 function(lagom_lib library fs_name)
     cmake_parse_arguments(LAGOM_LIBRARY "" "" "SOURCES;LIBS" ${ARGN})
     set(target_name "Lib${library}")
-    add_library(${target_name} ${LAGOM_LIBRARY_SOURCES})
+    if (CYGWIN)
```

Comment:
Don't do this, just tell people who are using Cygwin to `-DBUILD_SHARED_LIBS=OFF` on Lagom